app/mm-redux/types/users.ts

```diff
@@ -80,7 +80,19 @@ export type UserStatus = {
 	active_channel?: string;
 }
 
+export enum CustomStatusDuration {  // eslint-disable-line
```

Comment:
eslint shows error here - `'CustomStatusDuration' is already declared in the upper scope on line 83 column 13  no-shadow. To fix this, we have to add 
```
rules: {
  'no-shadow': 'off',
  '@typescript-eslint/no-shadow': 'error',
}
```

Is it okay to make this change? I thought that it can disable the rule for js files so that's why I didn't do it.

Reply:
yes it should be ok to make that change in the eslint rules